authors/apps/articles/models.py

```diff
@@ -37,3 +37,22 @@ def save(self, *args, **kwargs):
         if not self.slug:
             self.slug = get_unique_slug(self, self.title, 'title', 'slug')
         super().save(*args, **kwargs)
+
+
+class Ratings(models.Model):
+    ratings = models.DecimalField(
+        max_digits=5, decimal_places=0)
+    ratings_by = models.ForeignKey(
+        User,
+        related_name='ratings',
```

Comment:
Since we are using the profile now. I suggest we should stick to that in order to keep this principle in use.
![responsibility](https://user-images.githubusercontent.com/22131999/56459183-8c356880-6398-11e9-93f6-f444da2ac7f8.PNG)
Change from User model to the Profile model so that we can leave the User model for just authorization and authentication.

Reply:
I am only using the authentication as a reference for the user and the author

Reply:
I agree with @reifred.
Make the relationship between the profile of the author and the articles they've wrtten

Reply:
I have fixed this

Reply:
You need to generate a new migration file, @NicholusMuwonge.

Reply:
I have fixed this, thanks